continuous-integration/lint.sh

```diff
@@ -15,7 +15,8 @@ flake8 --config="$ROOT_DIR/.flake8" $pythonFiles || exit $?
 
 echo Running pylint on:
 echo "$pythonFiles"
-pylint -j 0 --rcfile "$ROOT_DIR/.pylintrc" --extension-pkg-whitelist=netCDF4 $pythonFiles || exit $?
+
+pylint -j 0 --rcfile "$ROOT_DIR/.pylintrc" --dummy-variables-rgx="((^|, )(app|rgb|xyz|contrast))+$" --extension-pkg-whitelist=netCDF4 $pythonFiles || exit $?
```

Comment:
Since this command has a lot of options now try splitting it in pieces for readability:
```
pylint \
    -j 0 \
    --rcfilr .. \
    --dummy-var .. \ 
etc
```

Reply:
done